bookstore/clone.py

```diff
@@ -299,3 +300,21 @@ def build_post_response_model(self, model, obj, s3_bucket, s3_object_key):
         if 'VersionId' in obj:
             model["versionID"] = obj['VersionId']
         return model
+
+
+def validate_relpath(relpath, settings):
+    """Validates that a relative path appropriately resolves given bookstore settings.
+    """
+    if relpath == '':
```

Comment:
The first two errors are server misconfigs. I would use a 501 and generically say. Server not configured for this. Details of why we think this is should be in server logs not response text.

Reply:
Is this a server misconfig when the relpath is going to be entered through the user's request parameter? This feels like a malformed request (and so, probably, 400)?

Reply:
I still will it to be less informative of a response and add logging on our side.

Reply:
You are right. For this one the 400 and error message are appropriate.